odoo/addons/base/static/src/scss/clearing_label.scss

```diff
@@ -0,0 +1,5 @@
+.o-autocomplete--dropdown-menu {
+    .o-autocomplete--dropdown-item [id^="clearing_label_id_"] span.text-muted {
+        float: right;
```

Comment:
you won't need this, see task https://www.odoo.com/odoo/project/133/tasks/5386833